src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -143,7 +143,11 @@ private void SetupCommonOptionsForCommands(List<Command> commands)
 
                 new Option(
                     "--include-non-security-rules",
-                    "Run all the rules against the templates, including non-security rules")
+                    "Run all the rules against the templates, including non-security rules"),
+
+                new Option<FileInfo>(
+                    "--custom-rules-json-path",
```

Comment:
Please rename to `--custom-json-rules-path` so it can't also be (falsely) interpreted as "the json path to custom rules".

Reply:
You'll need to update the variable name in code too.

Reply:
Done